src/libcore/benches/mem.rs

```diff
@@ -10,7 +10,6 @@
 
 use test::Bencher;
 
-// FIXME #13642 (these benchmarks should be in another place)
```

Comment:
I reopened this issue because I think it was closed prematurely. Please keep this FIXME for now (or fix #13642!).